data/src/main/java/in/koreatech/koin/data/api/auth/UserAuthApi.kt

```diff
@@ -87,4 +89,6 @@ interface UserAuthApi {
 
     @GET(URLConstant.OWNER.OWNER)
     suspend fun getOwnerTokenIsValid()
+    @POST("abtest/assign")
+    suspend fun postABTestAssign(@Body abTestRequest: ABTestRequest): ABTestResponse
 }
```

Comment:
사소하지만.. 공백 반영되면 좋을 거 같아요!
```suggestion

    @GET(URLConstant.OWNER.OWNER)
    suspend fun getOwnerTokenIsValid()
    
    @POST("abtest/assign")
    suspend fun postABTestAssign(@Body abTestRequest: ABTestRequest): ABTestResponse
}
```

Reply:
세세한 부분도 신경써보며 작업해야겠네요! 감사합니다!